server/scripts/add_teams.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+headers = {
+    'accept': 'application/json',
+    'Content-Type': 'application/json',
+}
+
+json_data = {
+    #id refers to team id, hunt_id refers to the hunt id
+    "id": "",
+    "hunt_id": "",
+    "name": "",
+    "teamLead": "",
+    "players":[],
+    "challengeResults": [], 
+    "invitations": []
+}
+
+def change(id, hunt_id, name, teamLead, players, challenges, invitation):
+    json_data['id'] = id
+    json_data['hunt_id'] = hunt_id
+    json_data['name'] = name
+    json_data['teamLead'] = teamLead
+    json_data['players'] = players
+    json_data['challengeResults'] = challenges
+    json_data['invitations'] = invitation
+
+change("Team1ID", "65f235aadc9df7de117ae2fc", "BEAMTEAM", "beamer", [], [], [])
```

Comment:
The players should not be an empty list, please add some players from the mock data so that they can populate on the screen when someone goes to the join team page.